### ISSUE TITLE
Rename nginx references to be more generic, as we now use nc not nginx

### ISSUE THREAD

**Issue by dfarrell07** [submariner-io/lighthouse#758]
Because of difficulty finding up-to-date, rootless nginx container images (required for OpenShift), we've moved to a much simpler nc-based "simpleserver". However, there are still very many references to nginx in the code, especially Lighthouse code. Ideally we'd rename these to be more generic, correct.

**Comment by ghost** [comment]
hey @dfarrell07 if this issue is still under progress. can I work on this ??

**Comment by skitt** [comment]
@mrmahi2003 the issue isn’t assigned to anyone, feel free to work on it.

**Comment by dfarrell07** [comment]
@mrmahi2003 Yes please! No one is working on it and we're looking for help with it.

**Comment by ghost** [comment]
glad for this  opportunity let me first look into this issue it would be better not to assign yet until I can confirm whether it is workable or not for me
